includes/openid-connect-generic-settings-page.php

```diff
@@ -347,24 +350,28 @@ private function get_settings_fields() {
 				'title'       => __( 'Link Existing Users', 'daggerhart-openid-connect-generic' ),
 				'description' => __( 'If a WordPress account already exists with the same identity as a newly-authenticated user over OpenID Connect, login as that user instead of generating an error.', 'daggerhart-openid-connect-generic' ),
 				'type'        => 'checkbox',
+				'disabled'    => defined( 'OIDC_LINK_EXISTING_USERS' ),
```

Comment:
@ChloeTigre you didn't document in your PR, and didn't add this to the PHPStan bootstap, adding this additional constant. Please remove this or add the missing documentation and additional support.